packages/satori-js-test/index.html

```diff
@@ -17,17 +17,21 @@
         .then(session => {
           _session = session;
           console.log("session", session);
-          client.sessionRefresh(session).then((session) => console.log("session refreshed, new token: ", session.token));
-          client.event(session, { name: "testEvent", value: "A value", metadata: { hello: "world" }}).then(() => console.log("sent event"));
-          client.events(session, [
-            { name: "sampleEventOne", value: "One", metadata: { meta: "data" }},
-            { name: "sampleEventTwo", value: "Two", metadata: { data: "meta" }}
-          ]).then(() => console.log("sent event"));
-          client.updateProperties(session, { email: "hello@world.com" }, { customProperty1: "A value here" });
-          client.listProperties(session).then((result) => console.log("properties:", result));
-          client.getExperiments(session, []).then((result) => console.log("experiments: ", result));
-          client.getFlags(session, []).then((result) => console.log("flags: ", result));
-          client.getLiveEvents(session, []).then((result) => console.log("live events:", result));
+          // client.getFlag(session, 'nonExistingFlagName', 'aDefaultValue').then((flag) => console.log('Non existing flag:', flag));
```

Comment:
How come these are commented out?

Reply:
Also in general we don't use the index.html for testing outside of a basic auth call.

Reply:
Removed all the function calls and kept it as a basic auth check.